hoomd/md/PotentialExternal.h

```diff
@@ -209,7 +239,10 @@ template<class T> void export_PotentialExternal(pybind11::module& m, const std::
     {
     pybind11::class_<T, ForceCompute, std::shared_ptr<T>>(m, name.c_str())
         .def(pybind11::init<std::shared_ptr<SystemDefinition>>())
-        .def("setParams", &T::setParams)
+        .def("setParams", &T::setParamsPython)
+        .def("getParams", &T::getParams)
+        .def("setE", &T::setParamsPython)
+        .def("getE", &T::getParams)
```

Comment:
I like this idea, but I have one suggestion. Instead of putting the template specializations in `module-md.cc`, I think it may be a better idea to put them in the header where the general template is defined for better organization. For example, that means I would put my template specialization in `PotentialExternal.h`, and we could move your Dipole specialization to `PotentialPairAniso.h`.